.travis.yml

```diff
@@ -1,47 +1,104 @@
 ---
```

Comment:
Please do a rebase against the master branch, because this file is managed by ```voxpupuli/modulesync```, ```voxpupuli/modulesync_config```.